Likhovodov/Simulated-Conversations
# Katie/symco 81

This ticket adds the "Delete and Notify" or just Delete when deleting a template! 
In order to test this please 👍 
1) create responses from a student
2) click delete and a modal should pop up and should tell you would you like to delete or delete and notify. the notify one sends an email to the student to re do a template and includes template name! i have attached screenshots of each new view

Modal from Individual response view
<img width="1118" alt="Screen Shot 2021-02-15 at 3 05 54 PM" src="https://user-images.githubusercontent.com/54516136/108000652-52871600-6f9f-11eb-9b86-f5a6990c9104.png">

modal for table view 
<img width="1124" alt="Screen Shot 2021-02-15 at 3 06 14 PM" src="https://user-images.githubusercontent.com/54516136/108000671-5adf5100-6f9f-11eb-927f-662f6e0b014f.png">

email sent: 
<img width="1030" alt="Screen Shot 2021-02-15 at 3 08 09 PM" src="https://user-images.githubusercontent.com/54516136/108000795-a0038300-6f9f-11eb-93c4-5ffa9f0d0c39.png">
